### DISCUSSION TITLE
Can the "ControlNet Inpaint" add the [Inpaint upload] as the img2img's inpaint had?

### DISCUSSION THREAD

**Discussion by Joe19CC** [lllyasviel/ControlNet#400] [Ideas]
Hi 
The new inpaint of controlnet is very powerful but its has only the basic drawing function comparing with the img2img's inpaint. There is an easy tool of inpaint called upload, which save time for drawing.
I do not know if there is same function for "inpaint upload" or not, or there is a similar way to use by controlnet?
Hoping this would be add in the future!
That would really save a lot of time and really precisly "Control the picture" than drawing.

**Comment by CanXu0728** [comment]
same, beg for "controlnet inpaint upload"

**Comment by Milor123** [comment]
We Need it guys ❤️  !!!

**Comment by geroldmeisinger** [comment]
I think this is something you have to request in the corresponding UI project (Automatic1111, ComfyUI)
